barcamp-theme/templates/index.html

```diff
@@ -7,11 +7,13 @@ <h2>Share Your Passion</h2>
         <h4>{{ SITENAME }}</h4>
         <span class="tab--tiny">With <a href="pages/faq.html">Lightning Talks!</a></span>
         <span class="tab--tiny">What is <a href="pages/about.html">Barcamp?</a></span>
-        <h5>May 2, 2020 from 9am-6pm<br />Kelley Engineering Center<br>
+        <!-- <h5>>May 2, 2020 from 9am-6pm<br />Kelley Engineering Center><br> -->
+        <p>
+        <a href="/bbc20-cancelled.html" id="register-button">Cancelled due to COVID-19</a>
```

Comment:
Add closing `<p>` tag

```suggestion
            <a href="/bbc20-cancelled.html" id="register-button">Cancelled due to COVID-19</a>
        </p>
```